assets/src/edit-story/components/checklist/designChecks.js

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+export function DesignChecks() {
+  return <div>{'design'}</div>;
```

Comment:
Does this need to be translated? If this is a placeholder, then maybe add a todo?

Reply:
cool. yeaup. I'll pull in the cards I'll be updating today here and if there are no cards in a category, I'll just return `null` and put a todo in there